esphome/components/bme68x_bsec2_i2c/bme68x_bsec2_i2c.h

```diff
@@ -0,0 +1,149 @@
+#pragma once
+
+#include "esphome/core/component.h"
+#include "esphome/components/sensor/sensor.h"
+#include "esphome/components/text_sensor/text_sensor.h"
+#include "esphome/components/i2c/i2c.h"
+#include "esphome/core/preferences.h"
+#include "esphome/core/defines.h"
+
+#include <cinttypes>
+#include <map>
+
+#ifdef USE_BSEC2
+#include <bsec2.h>
+#endif
+
+namespace esphome {
+namespace bme68x_bsec2_i2c {
+#ifdef USE_BSEC2
```

Comment:
This should just wrap everything
```suggestion

namespace esphome {
namespace bme68x_bsec2_i2c {
```